src/probe/probe.c

```diff
@@ -601,12 +601,12 @@ static int copy_from_pbuffer(struct probe_dma_buf *pbuf, void *data,
 /**
  * \brief Generate probe data packet header, update timestamp, calc crc
  *	  and copy data to probe buffer.
- * \param[in] buffer component buffer pointer.
+ * \param[in] buffer_id component buffer id
```

Comment:
@jsarha @abonislawski @kv2019i do you guys need the buffer * for IPC4 here or is ID all that's needed ?